src/cargo/ops/cargo_update.rs

```diff
@@ -787,20 +800,23 @@ struct PackageChange {
     package_id: PackageId,
     previous_id: Option<PackageId>,
     kind: PackageChangeKind,
+    is_member: Option<bool>,
 }
 
 impl PackageChange {
-    pub fn new(resolve: &Resolve) -> Vec<Self> {
+    pub fn new(ws: &Workspace<'_>, resolve: &Resolve) -> Vec<Self> {
         let diff = PackageDiff::new(resolve);
-        Self::with_diff(diff)
+        Self::with_diff(diff, ws)
     }
 
-    pub fn diff(previous_resolve: &Resolve, resolve: &Resolve) -> Vec<Self> {
+    pub fn diff(ws: &Workspace<'_>, previous_resolve: &Resolve, resolve: &Resolve) -> Vec<Self> {
         let diff = PackageDiff::diff(previous_resolve, resolve);
-        Self::with_diff(diff)
+        Self::with_diff(diff, ws)
     }
 
-    fn with_diff(diff: impl Iterator<Item = PackageDiff>) -> Vec<Self> {
+    fn with_diff(diff: impl Iterator<Item = PackageDiff>, ws: &Workspace<'_>) -> Vec<Self> {
+        let member_ids: HashSet<_> = ws.members().map(|p| p.package_id()).collect();
```

Comment:
Use `Workspace::is_member` instead?

Reply:
Thats annoying, it requires a `Package` just to do a lookup with `PackageId`